example/src/spa/view/not-found/not-found-view.js

```diff
@@ -0,0 +1,21 @@
+import './not-found.css';
+import ElementBuilder from '../../../classes/util/element/element-builder';
+import View from '../../../classes/view/view';
+
+const MainCssClasses = {
+    ERROR: 'not-found',
+};
+const TEXT_NOT_FOUND = 'Ошибка. Страница не найдена.';
+
+export default class NotFoundView extends View {
+    constructor() {
+        super();
+        this.htmlElement = this.createView();
```

Comment:
`super({tag: 'section', classNames: [MainCssClasses.ERROR], textContent: TEXT_NOT_FOUND})` in a good structure.